### PR TITLE
.golangci.yml: add `unused` linter

### PR DESCRIPTION
**Pull request by fyrchik**
Also fix critical bug regarding subnet removal.

### CODE REVIEW

File: pkg/innerring/subnet.go [hunk 1]
```diff
@@ -263,7 +263,7 @@ func (s *Server) handleSubnetCreation(e event.Event) {
 // catchSubnetRemoval catches event of subnet removal from listener and queues the processing.
 func (s *Server) catchSubnetRemoval(e event.Event) {
 	err := s.subnetHandler.workerPool.Submit(func() {
-		s.handleSubnetCreation(e)
+		s.handleSubnetRemoval(e)
```
Comment (alexvanin) [inline pkg/innerring/subnet.go:266]: ouch

File: .golangci.yml [hunk 2]
```diff
@@ -51,4 +52,4 @@ issues:
   exclude-rules:
     - path: policy/grammar\.go # ignore structtag issues there
       linters:
-        - govet
\ No newline at end of file
+        - govet
```
Comment (alexvanin) [inline .golangci.yml:55]: By the way, we can drop this exclusion rule. Code of the placement policy parser has been moved to SDK a while ago.

**Review by alexvanin — APPROVED** [review]

**Comment by alexvanin** [comment]
Also it would be nice to have `[#1123]` in all commit headers so all these changes can be visually grouped in commit history (cause we don't use merge commits).

File: .golangci.yml (hunk 2 above)
Comment (fyrchik) [reply .golangci.yml]: Fixed.

**Pull request merged**